src/database/seeders/20210120052931-userRoles.js

```diff
@@ -6,6 +6,12 @@ config();
 module.exports = {
   up: async (queryInterface, Sequelize) => {
     await queryInterface.bulkInsert('userRoles', [
+      {
```

Comment:
No need for seeding the MANAGER role, since we created him in database/managerRole.js.